.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report 🐛
+about: You're having technical issues
+labels: 'bug'
```

Comment:
'type: bug' would be better. You can imagine that we will have other labels like 'priority: p1' or others.

Reply:
HERE